openedx/core/djangoapps/user_api/accounts/utils.py

```diff
@@ -1,6 +1,7 @@
 """
 Utility methods for the account settings.
 """
+from __future__ import unicode_literals
```

Comment:
This is so the generate_password method in this file will create passwords as unicode and not as strings. Necessary for password normalization